File: CrudexBackend/CrudexBackend/Models/Forme.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace CrudexBackend.Models
{
    public class Forme
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int Hp { get; set; }
        public int Att { get; set; }
        public int Def { get; set; }
        public int SpA { get; set; }
        public int SpD { get; set; }
        public int Spe { get; set; }
        public string Type1 { get; set; }
        public string Type2 { get; set; }
        public string Image { get; set; }
        public string Description { get; set; }

        public int PokemonId { get; set; }
        public virtual Pokemon Pokemon { get; set; }
    }
}

File: CRUDex2/src/app/forme.ts
import { Pokemon } from './pokemon';

export class Forme {
  Id: number;
  Name: string;
  Hp: number;
  Att: number;
  Def: number;
  Spa: number;
  Spd: number;
  Spe: number;
  Type1: string;
  Type2: string;
  Image: string;
  Description: string;
  PokemonId: number;

  constructor(name: string,
              hp: number,
              att: number,
              def: number,
              spa: number,
              spd: number,
              spe: number,
              type1: string,
              type2: string,
              image: string,
              description: string,
              pokemonId: number) {
    this.Name = name;
    this.Hp = hp;
    this.Att = att;
    this.Def = def;
    this.Spa = spa;
    this.Spd = spd;
    this.Spe = spe;
    this.Type1 = type1;
    this.Type2 = type2;
    this.Description = description;
    this.PokemonId = pokemonId;
  }
}
